docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Text/RemoveDefaultStopWords.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Transforms.Text;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class RemoveDefaultStopWords
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create an empty data sample list. The 'RemoveDefaultStopWords' does not require training data as
+            // the estimator ('StopWordsRemovingEstimator') created by 'RemoveDefaultStopWords' API is not a trainable estimator.
```

Comment:
>StopWordsRemovingEstimator [](start = 31, length = 26)

any reason why you prefer this format other than `<see cref="StopWordsRemovingEstimator">` ?

Reply:
I doubt if it works in normal comments. <see cref="StopWordsRemovingEstimator">  works in xml comments.

---
In reply to: [271080226](https://github.com/dotnet/machinelearning/pull/3156#discussion_r271080226) [](ancestors = 271080226)